runtime/tests/interpreter/reference_test.go

```diff
@@ -452,7 +451,7 @@ func TestInterpretResourceReferenceAfterMove(t *testing.T) {
 
 		t.Parallel()
 
-		inter := parseCheckAndInterpret(t, `
```

Comment:
Given that these are interpreter tests, we only want to test semantically valid programs, and reference use after resource moves are not semantically valid anymore, maybe just remove this whole test case (`TestInterpretResourceReferenceAfterMove`), as long as we have equivalent tests for the checker, i.e. move test cases to the checker test cases if they cover cases not yet covered by the checker tests.

Same for the other test cases that assumed use after referenced resource move are valid, e.g. `TestInterpretReferenceUseAfterShiftStatementMove`

Reply:
Moved these to the checker tests: 55b8fa36662fbaeabee0e32b4bf6e528e07a2a8f

Reply:
Nice!